crates/cli/src/subcommands/describe.rs

```diff
@@ -13,6 +13,8 @@ pub fn cli() -> clap::Command {
         )
         .arg(
             Arg::new("entity_type")
+                .long("type")
+                .short('t')
```

Comment:
I think we might undo this change. This is required to use `entity_name`, and it's ignored unless we use `entity_name`.